common/app/common/LinkTo.scala

```diff
@@ -98,6 +98,7 @@ class CanonicalLink {
     "index",
     "page",
     "filterKeyEvents",
+    "automaticFilter",
```

Comment:
Do we need to add this here? My understanding was that linkto is used for keyboard navigators to jump to a section on the page but as we're rendering liveblogs through DCR. I'm not sure if we need to worry about this here - could be wrong about this though!

Reply:
Great point @abeddow91 I should've added the parameter within the `insignificantParams` which I just did. Do you think that's the correct place for it?

Reply:
Could you point me in the direction of it? I can't seem to spot this change

Reply:
Yes sure, it's here https://github.com/guardian/frontend/pull/25099/files#diff-8417633d0df6efd1f7447f7640712c26931ad19a09e74273fe9becfb382f946dR57